src-docs/src/views/suggest/hashtag_popover.js

```diff
@@ -30,6 +30,7 @@ export default (props) => {
       onClick={togglePopover}
       size="xs"
       iconType="arrowDown"
+      aria-label="hashtag button"
```

Comment:
We don't need to specify that a control is a button as that will be communicated by the screen reader.

Also, switching it to name what the button does, instead of the icon used.
```suggestion
      aria-label="Saved Queries popover"
```